Jenkinsfile

```
@@ -16,7 +16,7 @@
 
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value='pipeline-lib@your_branch') _
+// @Library(value='pipeline-lib@ryon-jensen/rpmlint_fix') _
```

We should put this back
```suggestion
// @Library(value='pipeline-lib@your_branch') _
```